event_questions.md

```diff
@@ -0,0 +1,70 @@
+# Selected questions and answers
+
+These are some selected Q&A during the live event we held, anonymized for privacy reasons.
+
+## What is the Github link to the repository?
+
+https://github.com/MLWorkshops/mlinterviews
+
+## Where are the setup instructions?
+
+https://github.com/MLWorkshops/mlinterviews
+
+## Regarding the number of jars.  Is there ramifications for resampling or is each sample not repeating?
+
+
+## How can I view the presentation slides?
+
+Presentation slides are located inside the individual directories (e.g. "stats_workshop/Slides.pdf"). They can either be viewed in Github or downloaded from Jupyter notebook to be viewed separately.
+
+## Can you recommend any good, concise resources for understanding statistics?
+
+* This is one of the best books on practical experimentation https://www.amazon.com/Field-Experiments-Design-Analysis-Interpretation/dp/0393979954/
+
+* https://camdavidsonpilon.github.io/Probabilistic-Programming-and-Bayesian-Methods-for-Hackers/
+
+## Any stats YouTube channels or podcasts you love?
+
+* StatQuest
+* 3 brown 1 blue
+* FiveThirtyEight
+* Two Minute Papers
+* Data Skeptic
+* Journal Club
+* Intro to Statistical Learning Book (YouTube videos)
+* https://www.oreilly.com/radar/topics/oreilly-data-show-podcast/
+* https://thedataexchange.media/author/bglorica/
+
+
+## I've read about model diagnostics/assumptions for linear models.  But what about logistic models?  I find I get stuck on multiple linear regression a lot when I get down to validating model assumptions and get stuck when dealing with outlier's (t student residuals, cooks distance)… I've read I'm supposed to maybe apply a different normalization or a different type of model (such as robust regression).
+
+Something that helped me was to think of assumptions less as “rules.” It’s not about whether you are “allowed” to use multiple linear regression when residuals look a certain way, but more about what those abnormalities are going to do to the results you’re trying to obtain.
+
+
+
+## If the class imbalance is 90:10 what should be done to train a good model? Also, what would be a good way to validate such a model?
+
+* Metrics for class imbalance :
+** AUC ROC
```

Comment:
Nice.  Maybe add two spaces at front of these two lines if trying for a bullet point list.